viewer/packages/camera-manager/src/types.ts

```diff
@@ -61,6 +61,20 @@ export interface RevealCameraControls extends THREE.EventDispatcher {
    */
   setScrollTarget: (target: THREE.Vector3) => void;
 }
+export type CameraState = {
+  /**
+   * Camera position in world space.
+   */
+  position?: THREE.Vector3;
+  /**
+   * Camera target in world space.
+   * **/
+  target?: THREE.Vector3;
+  /**
+   * Camera local rotation in quaternion form.
+   */
+  rotation?: THREE.Quaternion;
```

Comment:
Not sure how to handle this, BUT target and rotation are conflicting. What would it take to get rid of `target` and make that a purely internal thing in `CameraManager`-implementations?

Reply:
Alternatively, should we define a precedence order - e.g. that rotation takes precedence over target when both are set?

Reply:
That's how I implemented it in `DefaultCameraManager`. Target takes precedence when both are defined because I thought both of them are changing target, so explicit definition should have priority. Another option is to change rotation to only roll. That will remove the conflict as roll doesn't affect target-oriented rotation.

Reply:
Now, setting rotation and target at the same time throws an error